app/views/spree/admin/products/new.js.erb

```diff
@@ -0,0 +1,12 @@
+<%# This chunk is just a copy of Spree's core/app/views/spree/admin/products/new.js.erb %>
+$("#new_product").html('<%= escape_javascript(render :template => "spree/admin/products/new", :formats => [:html], :handlers => [:erb]) %>');
+handle_date_picker_fields();
+<% unless Rails.env.test? %>
+  $('.select2').select2();
+<% end %>
+$("#table-filter").hide();
+$("#admin_new_product").parent().hide();
```

Comment:
does somebody know how we could extend this Spree view without having to copy its contents?

Reply:
I tried using ruyb's `load` and `Deface` and I couldn't make any of them work. Not worth spending any more time on it.

Reply:
Yeah it seems an old `deface` issue 😞 https://github.com/spree/deface/issues/103